src/rust/engine/process_execution/src/lib.rs

```diff
@@ -353,12 +357,22 @@ impl TryFrom<MultiPlatformProcess> for Process {
 pub struct MultiPlatformProcess(pub BTreeMap<(PlatformConstraint, PlatformConstraint), Process>);
 
 impl MultiPlatformProcess {
-  pub fn user_facing_name(&self) -> Option<String> {
+  pub fn user_facing_name(&self) -> String {
+    self
+      .0
+      .iter()
+      .next()
+      .map(|(_platforms, process)| process.description.clone())
+      .unwrap_or_else(|| "<Unnamed process>".to_string())
```

Comment:
Note that the only reason this was `Option` before is that `next()` returns Option. But, in Python, we already validate that there is >=1 process.